ルビの修正 / id, title の整理(#20対応)

haruyama opened the pull request:
* test_ruby() を復活させました
     * エラー時に madb_id がある場合は 表示するようにしました
* docker-compose.yml に test service を追加しました
* テストが通るように ruby を直しました
    * ちょっと調べただけでは読み方がわからないものも多かったので 適当に入れたものもあります

見直しから WIP を取ります

haruyama committed:
Revive test_ruby()

haruyama committed:
Add test service

haruyama committed:
Fix invalid ruby

honeshabri commented on animedb.yml:
```diff
@@ -649646,7 +649646,7 @@
   title: ルパン三世 VS 名探偵コナン THE MOVIE
   suffix:
-  ruby: ルパン3セイ タイ メイタンテイコナン ザムービー
+  ruby: ルパンサンセイ タイ メイタンテイコナン ザムービー
```
✕「タイ」
◯「バーサス」
ガキ使限定CMの音声で確認

honeshabri commented on animedb.yml:
```diff
@@ -51056,7 +51056,7 @@
   title: みんなのうた「ショボクジラ・チビコブラ」
   suffix:
-  ruby: ミンナノウタ「ショボクジラ・チビコブラ」
+  ruby: ミンナノウタ ショボクジラ・チビコブラ
```
"・"が残っている

honeshabri commented on animedb.yml:
```diff
@@ -18169,7 +18169,7 @@
   title: みんなのうた「牧場の朝」
   suffix:
-  ruby: ミンナノウタ「ボクジョウノアサ」
+  ruby: ミンナノウタ ボクジョウノアサ
```
✕「ボクジョウ」
◯「マキバ」

honeshabri commented on animedb.yml:
```diff
@@ -16809,7 +16809,7 @@
   title: みんなのうた「ジロマジトンド・ジロトンド」
   suffix:
-  ruby: ミンナノウタ「ジロマジトンド・ジロトンド」
+  ruby: ミンナノウタ ジロマジトンド・ジロトンド
```
"・"が残っている

honeshabri commented on animedb.yml:
```diff
@@ -15829,7 +15829,7 @@
   title: みんなのうた「トム・ビリビ」
   suffix:
-  ruby: ミンナノウタ「トム・ビリビ」
+  ruby: ミンナノウタ トム・ビリビ
```
"・"が残っている

honeshabri commented on animedb.yml:
```diff
@@ -14809,7 +14809,7 @@
   title: エイトマン ロボット007 光線銃レーザー
   suffix: ダブり
-  ruby: エイトマン ロボット007 コウセンジュウレーザー
+  ruby: エイトマン ロボットゼロゼロセブン コウセンジュウレーザー
```
✕「ゼロゼロセブン」
◯「ゼロゼロナナ」
そういうキャラ名

honeshabri commented on animedb.yml:
```diff
@@ -14829,7 +14829,7 @@
   title: エイトマン ロボット007 光線銃レーザー
   suffix:
-  ruby: エイトマン ロボット007 コウセンジュウレーザー
+  ruby: エイトマン ロボットゼロゼロセブン コウセンジュウレーザー
```
✕「ゼロゼロセブン」
◯「ゼロゼロナナ」
こっちも

honeshabri commented on animedb.yml:
```diff
@@ -13024,7 +13024,7 @@
   title: みんなのうた「サンタはどこにいる?」
   suffix:
-  ruby: ミンナノウタ「サンタハドコニイル?」
+  ruby: ミンナノウタ サンタハドコニイル?
```
"?"が残っている

haruyama committed:
Fix invalid ruby

haruyama commented:
commit 漏れがありました.

バーサスとヴァーサスが混在してるので 統一してしまってもいいかもしれません

haruyama committed:
s/ゼロゼロセブン/ゼロゼロナナ/

haruyama committed:
Remove Trailing spaces

honeshabri commented:
> バーサスとヴァーサスが混在してるので 統一してしまってもいいかもしれません

『A-Ko The VS』は「ヴァーサス」と読むことになっていますので、下手に統一しないほうがいいです

haruyama commented:
`VS` 今回の変更範囲では「ヴァーサス」としていますが「バーサス」のほうがよいでしょうか

haruyama committed:
s/モンキーターンブィ/モンキーターンブイ/

haruyama committed:
s/イッキトウセン ジージー/イッキトウセン ダブルジー/

https://ja.wikipedia.org/wiki/%E4%B8%80%E9%A8%8E%E5%BD%93%E5%8D%83_(%E6%BC%AB%E7%94%BB)#.E4.B8.80.E9.A8.8E.E5.BD.93.E5.8D.83_Great_Guardians
> 読み方は「いっきとうせんグレートガーディアンズ」。略称は『一騎当千GG』で、読み方は「いっきとうせんダブルジー」。

haruyama commented:
気になった項目
- ANS000452200 
  - id: ギャラクシーエンジェル(第3期) / title: ギャラクシーエンジェル / ruby: ギャラクシーエンジェルエース ／ ギャラクシーエンジェルダブルエース(変更前)
  - ruby は ギャラクシーエンジェルダブルエース としました. id や title も ギャラクシーエンジェルAA にするのが適当かもしれません
- ANS000156600
  - id: アルフレッド・J・クワックの冒険 / title: アルフレッド・J・クワックの冒険 / ruby: チイサナアヒルノオオキナアイノモノガタリ アヒルノクワック
  - [小さなアヒルの大きな愛の物語 あひるのクワック - Wikipedia](https://ja.wikipedia.org/wiki/%E5%B0%8F%E3%81%95%E3%81%AA%E3%82%A2%E3%83%92%E3%83%AB%E3%81%AE%E5%A4%A7%E3%81%8D%E3%81%AA%E6%84%9B%E3%81%AE%E7%89%A9%E8%AA%9E_%E3%81%82%E3%81%B2%E3%82%8B%E3%81%AE%E3%82%AF%E3%83%AF%E3%83%83%E3%82%AF)
    - id, title を修正するのが適当かもしれません
- ANS000806400 
  - id: みなしごハッチ ママにだかれて / title: みなしごハッチ ママにだかれて / ruby: コンチュウモノガタリ ミナシゴハッチ ママニダカレテ
  - id, title, ruby で統一すべき?
- ANS000604400
  - id: それいけ!アンパンマン おともだちシリーズ/ファンタジー / title: それいけ!アンパンマン おともだちシリーズ/ファンタジー/ ruby: ソレイケアンパンマン オトモダチシリーズ ファンタジー バタコサントソラトブマフラー
  - [おいでよ！アンパンマン沼　バタコさんと空飛ぶマフラー](http://sorean.blog.fc2.com/blog-entry-865.html) アンパンまんの中の一話のようです. ruby にだけ バタコサントソラトブマフラー の情報があるので id, title に反映させたほうがいいかもしれません
- ANS008051700
  - id: ガリレイドンナ / title: ガリレイドンナ / ruby: ガリレイドンナ Storia di tre sorelle a caccia di un mistero (変更前)
  - Storia 以下はタイトルに含めなくてよいのではないかと思い ruby も ガリレイドンナ のみにしています

honeshabri commented:
> VS 今回の変更範囲では「ヴァーサス」としていますが「バーサス」のほうがよいでしょうか  

公式の資料（Webサイト、PVなど）で、特に読み方に触れてない限りは、「バーサス」に統一してください  
Googleで検索それぞれ検索すると  
- バーサス: 約 3,550,000 件  
- ヴァーサス: 約 1,760,000 件  

のように「バーサス」の方が一般的なので

builtinnya commented:
@haruyama 
ありがとうございます．
「気になった項目」については，本 PR で解決だと重いのであとで #20 から小分けするなりして対応します．

haruyama committed:
Use 'バーサス' for 'VS' ruby instead of 'ヴァーサス'

honeshabri commented:
> ギャラクシーエンジェル(第3期)   

Wikipediaによると  

> 映像作品タイトルは1 - 26話がギャラクシーエンジェルA（エース）、27 - 51話がギャラクシーエンジェルAA（ダブルエース）、54,55話がギャラクシーエンジェルS（エス）。  

どれか特定の名前をつけるよりも第3期でくくってしまった方がいいです。  

なおANS000516000の場合は第8話まで『流星戦隊ムスメット』で第9話 - 第11話の間『流星戦隊オトメット』となり、第12話で『帰ってきた 流星戦隊ムスメット』となりましたが、基本は『流星戦隊ムスメット』なので『流星戦隊ムスメット』に統一します。

haruyama commented:
このPR内の変更について VS の ruby として 「バーサス」 を用いるようにしました.
ただし,

```
- madb_id: ANS000640600
  id: ルパン三世VS名探偵コナン
  title: ルパン三世VS名探偵コナン
  ruby(変更前): ルパンサンセイ ヴァーサス メイタンテイコナン ／ ルパンサンセイ ブイエス メイタンテイコナン ／ ルパンサンセイ タイ メイタンテイコナン
```

については 元の ruby に ヴァーサスがあるので `ruby: ルパンサンセイ ヴァーサス メイタンテイコナン` としています. ここもバーサスにしたほうがよければそのようにします

haruyama commented:
> ギャラクシーエンジェル(第3期)

ruby としては `ギャラクシーエンジェル` でしょうか `ギャラクシーエンジェル ダイサンキ` でしょうか?

ANS000516000 については `流星戦隊ムスメット` の id, title, ruby だけ残せばいいでしょうか?

honeshabri commented:
> ギャラクシーエンジェル(第3期)  

rubyは`ギャラクシーエンジェル`でお願いします  
rubyはtitleの読み仮名を書く方針です
idはtitle + suffixで構成されていますが、suffixは識別のためにこちらで勝手に付けたものですので

honeshabri commented:
> ANS000516000 については`流星戦隊ムスメット`の id, title, ruby だけ残せばいいでしょうか?  

それでお願いします

haruyama committed:
Set ギャラクシーエンジェル to ruby of id:ギャラクシーエンジェル(第3期)

haruyama committed:
Set 流星戦隊ムスメット to id and title of ANS000516000

Set "リュウセイセンタイ ムスメット" to ruby of ANS000516000

honeshabri commented:
> ルパン三世VS名探偵コナン

「ヴァーサス」でお願いします
調べたところamazonの内容紹介に次のような文があります

> “決着(ケリ)つけようぜー。"今世紀最大の宿命の対決(ヴァーサス)!!

haruyama committed:
Set あひるのクワック to id and title of ANS000156600

Set アヒルノクワック to ruby of ANS000156600

haruyama committed:
Add 昆虫物語 prefix to id and title of みなしごハッチ series

ruby に コンチュウモノガタリ があって id, title に
昆虫物語がないものに追加

haruyama committed:
Change id and title of ANS000604400

それいけ!アンパンマン おともだちシリーズ/ファンタジー
->
それいけ!アンパンマン おともだちシリーズ/ファンタジー「バタコさんと空飛ぶマフラー」

haruyama commented:
`id`，`title`，`ruby` のミスマッチが起きている項目が存在する #20 
のコメントをみて commit を足してしまいました. PR を ruby だけに限定したほうがよければ PR を作り直します.

builtinnya commented:
> id，title，ruby のミスマッチが起きている項目が存在する #20 
> のコメントをみて commit を足してしまいました. PR を ruby だけに限定したほうがよければ PR を作り直します.

構いません．ありがとうございます．
こちら WIP でしょうか？そうでなければマージしようと思います．

haruyama commented:
ルパン三世 VS 名探偵コナン THE MOVIE も `ヴァーサス` が適当と思うので変更します.
[ルパン三世VS名探偵コナン THE MOVIE - Wikipedia](https://ja.wikipedia.org/wiki/%E3%83%AB%E3%83%91%E3%83%B3%E4%B8%89%E4%B8%96VS%E5%90%8D%E6%8E%A2%E5%81%B5%E3%82%B3%E3%83%8A%E3%83%B3_THE_MOVIE) でも ヴァーサスですね

これを push  したら WIP をはずします

haruyama committed:
Change ruby of ANS008105200

ルパンサンセイ バーサス メイタンテイコナン ザムービー
->
ルパンサンセイ ヴァーサス メイタンテイコナン ザムービー

builtinnya commented:
@haruyama 
長丁場お疲れ様です．
大変助かりました．ありがとうございます 🙇 
では，マージいたします
